soc/renesas/ra/ra8d1/sections.ld

```diff
@@ -77,3 +77,11 @@ SECTION_PROLOGUE(.option_setting_s,,)
 	KEEP(*(.option_setting_bps_sel3))
 	__OPTION_SETTING_S_End = .;
 } GROUP_LINK_IN(OPTION_SETTING_S) = 0xFF
+
+SECTION_DATA_PROLOGUE(.sdram,(NOLOAD),)
+{
+	__SDRAM_Start = .;
+	KEEP(*(.sdram*))
+    KEEP(*(.lvgl_buf*))
```

Comment:
Not sure this should be at the SOC level. Will every developer using this SOC want to link LVGL buffers into SDRAM? Perhaps they are driving a small display, and their board has no SDRAM. I think this should be at the board level, like we do for the MIMXRT595 for example: https://github.com/zephyrproject-rtos/zephyr/blob/5dc5fa5ee2ac4c0b3914ce926d1e126f052d1e99/boards/nxp/mimxrt595_evk/dc_ram.ld#L14

Reply:
Done. I moved it into board layer